### PR TITLE
Show more button styling

### PR DESCRIPTION
**Pull request by whitzhu**
## Summary
screen shot 2016-07-14 at 1 08 10 pm

*Short description*

## TODO

- [ ] Have tests been written for the new code?
- [ ] Has documentation been written/updated?
- [ ] New dependencies (if any) added to requirements file
- [ ] Add an entry to CHANGELOG.rst
- [ ] Add yourself it AUTHORS.rst if you don't appear there

## Reviewer guidance

*If you PR has a significant size, give the reviewer some helpful remarks*

## Issues addressed

List the issues solved or partly solved by the PR

## Documentation

If the PR has documentation, link the file here (either .rst in your repo or if built on Read The Docs)

## Screenshots (if appropriate)

They're nice. :)

### CODE REVIEW

**Comment by whitzhu** [comment]
Hey @indirectlylit ! I updated the styling for the show more button. I adjusted the button styling that's inside 'core-global.styl' not sure if that was okay or if 'button' might be reused elsewhere.

File: kolibri/plugins/learn/assets/src/vue/learn-page/index.vue [hunk 1]
```diff
@@ -13,10 +13,10 @@
 
   <div class='button-wrapper'>
     <button @click='toggle()' v-if='expanded'>
-      <i class="material-icons">&#xE5CE;</i> Show Less
+      <i class="material-icons">&#xE5CE;</i> <span class="button-text">Show Less</span>
     </button>
     <button class='pure-button' @click='toggle()' v-else>
-      <i class="material-icons">&#xE5CF;</i> <span>Show More</span>
+      <i class="material-icons">&#xE5CF;</i> <span class="button-text">Show More </span>
```
Comment (indirectlylit) [inline kolibri/plugins/learn/assets/src/vue/learn-page/index.vue]: looks like these new `button-text` classes are not used

Comment (indirectlylit) [inline kolibri/plugins/learn/assets/src/vue/learn-page/index.vue]: oh - I didn't see that in the global styles. Hmm I don't love the idea of putting that class there... I need to think about that a moment

**Comment by indirectlylit** [comment]
hey, I made a PR to refactor this a bit: https://github.com/whitzhu/kolibri/pull/3

**Comment by indirectlylit** [comment]
conflict will probably need to be resolved with: https://github.com/learningequality/kolibri/pull/259

File: kolibri/plugins/learn/assets/src/vue/learn-page/index.vue (hunk 1 above)
Comment (rtibbles) [inline kolibri/plugins/learn/assets/src/vue/learn-page/index.vue]: Also, note that this now conflicts with master, after changes from my PR.

**Pull request merged**